soc/wch/ch32v/Kconfig

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Michael Hope
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_CH32V
+	select RISCV
+	select BUILD_OUTPUT_HEX
+	select CH32V00X_SYSTICK
+	select ATOMIC_OPERATIONS_C
+	imply XIP
+
+if SOC_FAMILY_CH32V
+
+config VECTOR_TABLE_SIZE
+	int "Number of Interrupt and Exception Vectors"
+	range 20 256
+	help
+		This option defines the total number of interrupt and exception
+		vectors in the vector table.
```

```suggestion
	  This option defines the total number of interrupt and exception
	  vectors in the vector table.
```

Help string should be indented with 2 spaces

---

Done. Thank you.